saleor/graphql/attribute/tests/test_utils.py

```diff
@@ -1,15 +1,25 @@
+from unittest.mock import Mock, patch
+
 import graphene
 import pytest
 from django.conf import settings
 from django.core.exceptions import ValidationError
 
+from saleor.permission.enums import (
```

Comment:
The rest of the imports are relative. Can we convert it to relative also, to match the import way in the file?